src/tests/dotnet-trace/ChildProcessTests.cs

```diff
@@ -12,9 +12,12 @@
 using Xunit.Extensions;
 using TestRunner = Microsoft.Diagnostics.CommonTestRunner.TestRunner;
 
+// Newer SDKs flag MemberData(nameof(Configurations)) with this error
+// Avoid unnecessary zero-length array allocations.  Use Array.Empty<object>() instead.
+#pragma warning disable CA1825 
```

Comment:
Would it be easier for all the tests if we did a warning disable in the project files? Or add `<NoWarn>1825</NoWarn>` to https://github.com/dotnet/diagnostics/blob/main/src/tests/Directory.Build.props

Reply:
Doesn't work when building from VS.